### PR TITLE
Add Postman collection

### PR DESCRIPTION
**Pull request by ritawatson**
Added a Postman collection doc for https://spectrocloud.atlassian.net/browse/DOC-398.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/981#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @ritawatson. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by ritawatson** [comment]
I'm not sure about the step outcome (after step 3) because I'm not using a downloaded version of Postman. Could you please verify?

File: content/api/3-postman-collection.md
Comment (karl-cardenas-coding) [inline content/api/3-postman-collection.md:22]: We can lump the following paragraph + info box into a prerequisites section. We also should add a bullet specifying Postman App vX.X.X or greater.

File: content/api/3-postman-collection.md
Comment (karl-cardenas-coding) [inline content/api/3-postman-collection.md:25]:
```suggestion
* An authentication credential. 
```

File: content/api/3-postman-collection.md
Comment (karl-cardenas-coding) [inline content/api/3-postman-collection.md:43]:
```suggestion
Use the following URL to import your configuration into Postman:
```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

**Comment by ritawatson** [comment]
Comments are incorporated. The current state of the doc looks as follows:
CleanShot 2022-11-14 at 10 52 59

File: content/api/3-postman-collection.md
Comment (karl-cardenas-coding) [inline content/api/3-postman-collection.md:26]:
```suggestion
* [Postman application](https://www.postman.com/downloads/) v10.2.2+
```
Let's add the link to Postman. All prerequisites should contain a link to the resource.

File: content/api/3-postman-collection.md
Comment (karl-cardenas-coding) [inline content/api/3-postman-collection.md:49]:
```suggestion
<br />
```
This adds a linebreak between the info box and the list.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

**Comment by ritawatson** [comment]
Committed suggestions:
- line break after info box
- changes to Postman version

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/981#pullrequestreview-1179844400" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/981#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: Cannot trigger testing until a trusted user reviews the PR and leaves an `/ok-to-test` message.

<details>

In response to [this](https://github.com/spectrocloud/librarium/pull/981#issuecomment-1314379535):

>/test librarium_release_build


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
